operations/observability/Observability.md

```diff
@@ -0,0 +1,357 @@
+# Install and Upgrade Observability Framework
+
+## Overview
+
+The Install and Upgrade Observability Framework includes Goss tests health check results in visual Grafana dashboard providing key insights into the health of an HPC from the simple components to the functional and aggregate view of health.
+
+The automated Time to Install(TTI) and Time to upgrade(TTU) provides us the time and other important details for each portion of the HPC install like when and on what machine a given command was run.
+This allows us to dynamically capture and store in a file how long the commands/scripts were running, as well what inactivity time was there from the user, as well as how much debug time.
+
+Instead of remembering to notate and record in excel and confluence pages the time spend in actual work, debug, wait time and total time at multiple parts of install for all the product streams, the automated observability time tracking will do this for us.
```

Comment:
```suggestion
The automated observability time tracking will record the time spent in actual work, debug, wait time, and total time at multiple parts of install for all the product streams.
```
If it is important to still call out the replacement of Excel and Confluence pages, I would list that after the functionality of the tool.

Reply:
sure. changed.